exercises/isbn-verifier/description.md

```diff
@@ -5,21 +5,21 @@ numbers. These normally contain dashes and look like: `3-598-21508-8`
 
 ## ISBN
 
-The ISBN-10 format is 9 digits (0 to 9) plus one check character (either a digit or an X only). In the case the check character is an X, this represents the value '10'. These may be communicated with or without hyphens, and can be checked for their validity by the following formula:
+The ISBN-10 format is 9 digits (0 to 9, represented below with a subscript) plus one check character (either a digit or an X only). In the case the check character is an X, this represents the value '10'. These may be communicated with or without hyphens, and can be checked for their validity by the following formula:
+
+
+> (1₁ * 10 + 2₂ * 9 + 3₃ * 8 + 4₄ * 7 + 5₅ * 6 + 6₆ * 5 + 7₇ * 4 + 8₈ * 3 + 9₉ * 2 + 10₁₀ * 1) mod 11 == 0
```

Comment:
This notation is really confusing. I only know digits with indices like this where the index clarifies which base the number is in. I think it was suggested that these should be `d` with subscripts?